gammapy/image/core.py

```diff
@@ -888,7 +888,13 @@ def plot(self, ax=None, fig=None, add_cbar=False, stretch='linear', **kwargs):
         fig : `~matplotlib.figure.Figure`, optional
             Figure
         stretch : str, optional
-            Scaling for image ('linear', 'sqrt', 'log')
+            Scaling for image ('linear', 'sqrt', 'log').
+            Similar to normalize and stretch functions in ds9.
```

Comment:
@facero Thanks for adding this example! Just one additional comment:

Could you move this example into a new "examples" sections of the `SkyImage.plot()` docstring? 
This has the advantage that the code will be formatted nicely and users can directly copy and paste from the docs. (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/image/core.py#L1024) how this section is defined...). Ideally the example you give should work, when just copied and pasted in a notebook or IPython shell. Maybe you can extend the example to achieve this. You could e.g. use a Fermi-LAT example image (as done  [here](https://github.com/gammapy/gammapy/blob/master/docs/image/index.rst#getting-started)).

Reply:
@adonath Done. 
I added a standalone example. Tested it and it worked for me. Let me know if this is ok.